src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -147,15 +146,20 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     // Remove all but basic styling HTML and img tags from the message content
     Document.OutputSettings settings = new Document.OutputSettings();
     settings.prettyPrint(false);
-    String cleanedMessageContent = Jsoup.clean(messageContent, "",
+    messageContent = Jsoup.clean(messageContent, "",
         Whitelist.basicWithImages(), settings);
 
```

Comment:
I'm fine with either way, but maybe it would be good to keep the original message content, and set the cleaned version in a separate variable (like how it was before!)